src/report_table.js

```diff
@@ -527,7 +526,7 @@ const buildReportTable = function (
       );
   };
 
-  renderTable().then(() => {
+  await renderTable().then(() => {
```

Comment:
There should no "then()"s here in an async function. Mixing async syntax and promise chaining makes it hard to follow and debug.

It should be:
```
await renderTable()
...
try {
    await buildreporttable();
  } catch(err) {
    console.error
  } finally {
    done()
  }